project/src/app/data/product_data.ts

```diff
@@ -0,0 +1,22 @@
+export let data = [
```

Comment:
1) "let" change to "const"
2) change product_data.ts to products.ts
3) change the folder name to "mock" or you can leave the name "data"